Develappers.BillomatNet/Helpers/InvoiceMappingExtensions.cs

```diff
@@ -403,13 +403,12 @@ internal static Api.Invoice ToApi(this Invoice value)
 
             var paymentTypes = string.Join(",", value.PaymentTypes);
 
-            var apiInvoiceItems = new List<Api.InvoiceItem>();
-            foreach (var item in value.InvoiceItems)
+            if (value.InvoiceItems == null)
```

Comment:
im Fall von null keine Exception werfen, sondern einfach die items ignorieren - dann wird eine Rechnung angelegt, aber ohne Items - was ja in der API auch möglich ist